### DISCUSSION TITLE
unable to join Slack channel, "This link is no longer active"

### DISCUSSION THREAD

**Discussion by UglySoftware** [electricitymaps/electricitymaps-contrib#6366] [General]
Hi all,

I'm having trouble joining the Slack channel.  There are several places in the wiki and on GitHub that point to https://slack.electricitymaps.com/, which displays a Join button.  However, when I click the Join button I always get the error message "This link is no longer active   To join this workspace, you’ll need to ask the person who originally invited you for a new link.".

Does anyone know, is the Slack channel still active?  If so, how do I sign up?

I've heard that Slack invite links expire every 30 days so I wonder if it's just an old link.

Thanks!

**Comment by VIKTORVAV99** [comment]
@madsnedergaard do you know what might be the issue?

**Comment by madsnedergaard** [comment]
Sorry for that, the link expired - I have updated it and it should work now :)
